applications/ssal-commods-dex.md

```diff
@@ -0,0 +1,239 @@
+# Ssal: Ink Commodities Exchange
+- **Team Name:** Mansa Capital
+- **Product Name:** Ssal
+- **Payment Address:** 15JQDAHWTbWju9RWQfP7EQvNV9skCvm5xh69Mb5J5YMxY8Hm (USDT)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview :page_facing_up:
+
+Ssal, a blockchain-based commodities exchange, is a new version of our product called Riso, which we are now adapting to run in the Polkadot ecosystem using smart contracts. Since Riso will be mentioned frequently throughout the document, it is important to keep in mind that Ssal is a smart contract based, Substrate-specific version of Riso: Riso is a precursor to Ssal. 
+
+### Overview
+
+Ssal is a blockchain-based commodities exchange platform which leverages the power of distributed ledger technology to address the challenge of transparency and trust in commodity trading while also lowering the barrier to entry for producers and consumers to enter the market. The platform will target burgeoning markets that do not have the advantage of a commodities exchange to hedge crop prices and/or cannot trade at the volumes which traditional exchanges require. 
+
+We built the original version of this project, Riso, on Substrate, but realized that architecting a fully fledged parachain goes out of scope for the purposes of our application. Our team is looking for a grant to build Ssal using smart contracts running on Substrate-based chains. We believe that the technology is ready to be applied to a practical, real-world application. We intend to use the grant money to travel on-site in Ivory Coast this winter and set up a trial exchange between a local farmer and distributor whom we have been in contact with. In short, we would like to provide the ecosystem with a user-facing, commercial product, and hopefully, by documenting our process, we can inspire others to do the same.
```

Comment:
The grants program doesn't fund traveling. Please see our [FAQ](https://grants.web3.foundation/docs/faq#what-activitiespositions-do-you-fund) and [application guidelines](https://grants.web3.foundation/docs/Introduction/intro) for details on what we fund.